src/Providers/Stripe/Exception.php

```diff
@@ -2,22 +2,20 @@
 
 namespace PodPoint\Payments\Providers\Stripe;
 
-use Stripe\PaymentIntent;
-
 class Exception extends \Exception
 {
     /**
      * The response from the Stripe API.
      *
-     * @var PaymentIntent
+     * @var mixed
      */
     private $response;
 
     /**
-     * @param PaymentIntent $response
+     * @param mixed $response
      * @param \Throwable|null $previous
      */
-    public function __construct(PaymentIntent $response, \Throwable $previous = null)
```

Comment:
it was quite good to have this typehinted - I think a different exception should be used if this is going to be a different type